CONTRIBUTING.rst

```diff
@@ -151,8 +151,8 @@ but here is a simple overview:
 
     $ git clone git@github.com:YOUR_GITHUB_USERNAME/pytest.git
     $ cd pytest
-    $ git checkout pytest-2.7      # if you want to fix a bug for the pytest-2.7 series
-    $ git checkout master          # if you want to add a feature bound for the next minor release
+    $ git checkout 2.8.X   # if you want to fix a bug for the pytest-2.8 series
+    $ git checkout master  # if you want to add a feature bound for the next minor release
```

Comment:
Aside from my comment, I think it is clearer to show the examples using a single command form:

```
$ git checkout -b your-branch-name 2.8.X   # if you want to fix a bug for the pytest-2.8 series
$ git checkout -b your-branch-name master  # if you want to add a feature bound for the next minor release
```